sdf_timing/tests/data/golden/clb.sdf

```diff
@@ -6,9 +6,15 @@
         (CELLTYPE "LUT_OR_MEM5LRAM")
         (INSTANCE SLICEM)
         (TIMINGCHECK
+                (HOLD CLK (posedge CLK) (0.507::0.411))
+                (HOLD CLK (posedge CLK) (0.728::0.538))
+                (HOLD CLK (posedge CLK) (0.579::0.46))
```

Comment:
why do we have 4 entries for the same timing? Shouldn't this be handled by speed corners?

Reply:
The duplicate arcs are due to the fact they are in the input `clb.sdf` file. The original parser code removed these duplicate entries so that it kept the last record. A parser shall not do that as it would be then losing information it provides to downstream SW layers. Think that you would like to do pretty printing or write a syntax aware editor; it would not be possible if the parser is removing syntax elements.

So if you compare the output SDF to `clb.sdf` input, you would notice that now they are the same. The only difference is that `sdfparse.emit()` sorts the parse elements so they appear reordered in the output file. I did not fix `emit()` to not do sorting. Instead, I created a new output `sdfwrite.print_sdf()` that writes out elements in the order of the input. To see the difference, compare the outputs of `utils/sdfwriter.py` and `utils/reformat.py`. For example:

    python3 utils/sdfwriter.py --stdout sdf_timing/tests/data/clb.sdf
    python3 utils/reformat.py --stdout sdf_timing/tests/data/clb.sdf